tools/UpdatePaths/Scripts/73781_sniper_automatic_to_rifle.txt

```diff
@@ -1,3 +1,3 @@
 #comment This repaths sniper rifles from the automatic subtype to the rifle subtype.
 
-/obj/item/weapon/automatic/sniper : /obj/item/weapon/rifle/sniper{@OLD}
+/obj/item/gun/ballistic/automatic/sniper : /obj/item/gun/ballistic/rifle/sniper{@OLD}
```

Comment:
Why are we also touching this old script?

Reply:
because it's not correct. I made that script